consent-ui/src/app/ais/initiation/consent-initiate/consent-initiate.component.ts

```diff
@@ -1,13 +1,13 @@
-import { Component, OnInit } from '@angular/core';
+import {Component, OnInit} from '@angular/core';
 import {combineLatest} from "rxjs";
```

Comment:
no need to use explicidly combineLatest operator, since only queryParams are Observabe, but path params can be accessed directly